packages/survey/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@easyfeedback/survey",
+  "version": "0.0.0",
+  "description": "A collection of component for surveys",
+  "keywords": [
+    "easyfeedback",
+    "survey"
+  ],
+  "homepage": "https://github.com/easyfeedback/RCL#readme",
+  "bugs": {
+    "url": "https://github.com/easyfeedback/RCL/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/easyfeedback/RCL.git"
+  },
+  "license": "MIT",
+  "author": "Jury Giannelli",
+  "exports": {
+    ".": {
+      "require": "./dist/cjs/index.js",
+      "default": "./dist/esm/index.js"
+    }
+  },
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "typings": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "concurrently yarn:build:*",
+    "build:esm": "cross-env BABEL_ENV=esm babel src --root-mode upward --extensions .ts,.tsx -d dist/esm --source-maps",
+    "build:cjs": "cross-env BABEL_ENV=cjs babel src --root-mode upward --extensions .ts,.tsx -d dist/cjs --source-maps",
+    "build:types": "tsc --emitDeclarationOnly --declaration --declarationDir dist/types",
+    "lint": "concurrently yarn:lint:*",
+    "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx --ignore-path ../../config/.eslintignore",
+    "lint:types": "tsc --noEmit",
+    "test": "jest --env=jsdom --passWithNoTests",
+    "test:ci": "jest --runInBand",
+    "test:ci:cov": "yarn test:ci --coverage",
+    "test:cov": "yarn test --coverage",
+    "test:watch": "yarn test --watch -u"
+  },
+  "dependencies": {
+    "@easyfeedback/test-utils": "^1.1.7",
+    "uuid": "^8.3.2"
```

Comment:
You use explicit `@chakra-ui/theme-tools` and `react-icons`. So they heave to add to the dependencies.
They will work in this monorepo, but not as standalone package.

Reply:
I need to configure vsc better, so it will work and check issues on monorepo :/

Reply:
I am afraid that this is possible. But maybe you can find a solution for this.

Basically all explicit import calls must always be added in the `package.json` of the corresponding package either in the `peerDependencies` or `dependencies`.

Reply:
deps added!